.github/workflows/dotnet.yml

```diff
@@ -12,14 +12,24 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: windows-latest
```

Comment:
Switching to windows CI so we can test MAUI build for Android, Windows, iOS and macOS